test/actions/test_images_error.ts

```
@@ -0,0 +1,105 @@
+import test from "node:test";
+// import assert from "node:assert";
+import { GraphAILogger } from "graphai";
+
+import { getFileObject } from "../../src/cli/helpers.js";
+import { createStudioData } from "../../src/utils/context.js";
+import { images, generateBeatImage } from "../../src/actions/images.js";
```

ESLint

[error] Remove this unused import of 'generateBeatImage'.

(sonarjs/unused-import)